code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml.cs

```diff
@@ -4,8 +4,10 @@ namespace Param_RootNamespace;
 
 public sealed partial class MainWindow : WindowEx
 {
-    public Microsoft.UI.Dispatching.DispatcherQueue dispatcherQueue;
-    private UISettings _settings;
+    private Microsoft.UI.Dispatching.DispatcherQueue dQueue;
```

Comment:
can this name change be a breaking change if the old name was used already?

Reply:
oh, it is not a breaking change. I committed last change recently.